proknow/Patients/Entities/EntitySummary.py

```diff
@@ -68,17 +70,56 @@ def get(self):
                 entities = [entity.get() for entity in patient.find_entities(lambda entity: True)]
         """
         entity_type = self._data["type"]
+        DELAY = 0.2
+        MAX_COUNT = 5 / DELAY
         if entity_type == "image_set":
-            _, image_set = self._requestor.get('/workspaces/' + self._workspace_id + '/imagesets/' + self._id)
+            count = 0
+            while True:
+                _, image_set = self._requestor.get('/workspaces/' + self._workspace_id + '/imagesets/' + self._id)
+                if image_set["status"] == 'completed':
+                    break
```

Comment:
Can I suggest that instead of using `while True` and `break`, you declare a variable `status = ''` outside of the `if` condition (i.e., as a sibling of `count`) and then all of these `while True` statements can change to something like:
```
while status != 'completed':
    _, image_set = self._requestor.get('/workspaces/' + self._workspace_id + '/imagesets/' + self._id)
    status = image_set["status"]
    if status != 'completed': # pragma: no cover (should not occur in normal circumstances)
        if count < MAX_COUNT:
            time.sleep(DELAY)
            count += 1
        else:
            raise TimeoutExceededError('Timeout exceeded while waiting for image set entity to reach completed status')
```
Something about having an infinite loop (even with a break) gives me the heebie jeebies, even if they are more-or-less equivalent.

Reply:
See https://www.python.org/dev/peps/pep-0315/

Reply:
> Users of the language are advised to use the while-True form with an inner if-break when a do-while loop would have been appropriate.

Reply:
OKay.